stew/base36.nim

```diff
@@ -0,0 +1,238 @@
+## Copyright (c) 2025 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+## This module implements Base36 encoding and decoding procedures.
+## This module supports two variants of Base36 encoding (Bitcoin and Flickr).
```

Comment:
I've been trying to find references for this point without success so far. Both seem to use Base58, not Base36?
- https://metacpan.org/pod/Encode::Base58 specifically describes itself as being "for Flickr short URI" purposes; and
- https://en.bitcoin.it/wiki/Base58Check_encoding

It would be useful to cite, via URLs, in the comment block, where this statement is coming from.

Reply:
Oh. It's because you just copy/pasted this entire module from https://github.com/status-im/nim-stew/blob/master/stew/base58.nim

Reply:
Yup that's it